### PR TITLE
Foundation update to scripts to remove manual step from deployments

### PR DESCRIPTION
**Pull request by stephenwf**
@jamesgorrie

### CODE REVIEW

File: grunt/config/copy.js [hunk 1]
```diff
@@ -2,16 +2,29 @@
...
+    src: ['quilt.html'],
+    dest: 'quilt/index.html',
+  },
+  quilt_assets: {
+    expand: true,
+    cwd: 'dist/assets',
+    src: ['**'],
+    dest: 'dist/quilt/assets',
+  },
+};
```
Comment (stephenwf) [inline grunt/config/copy.js:30]: This is the main change here. Didn't realise how big the lock file was

https://github.com/wellcometrust/annotation-viewer/pull/11/files/eedaf79fe13c79126d51c61d0865f64825083981#diff-cd6837564890516d69cfe599f83b6a8e

File: quilt/index.html/quilt.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        }
+      </style>
+    </div>
+    <script type="application/javascript">
+      window.wellcome_application = 'quilt';
+      window.wellcome_page = 'quilt';
+    </script>
+    <script src="./assets/js/script.js"></script>
+  </body>
+</html>
```
Comment (jamesgorrie) [inline quilt/index.html/quilt.html:32]: This is a weird filename no?

File: grunt/tasks/dist.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 // grunt/tasks/dist.js
 
 module.exports = function(grunt) {
-  grunt.registerTask('dist', ['assemble:dist', 'scsslint', 'sass:dist','postcss:dist', 'browserify:dist', 'accessibility', 'groc', 'modernizr:dist', 'copy:dist', 'browserSync:dist']);
-};
\ No newline at end of file
+  grunt.registerTask('dist', ['assemble:dist', 'scsslint', 'sass:dist','postcss:dist', 'browserify:dist', /*'accessibility',*/ 'groc', 'modernizr:dist', 'copy:dist', 'copy:quilt_file', 'copy:quilt_assets', 'browserSync:dist']);
```
Comment (jamesgorrie) [inline grunt/tasks/dist.js:4]: Where'd accessibility go?

**Review by jamesgorrie — APPROVED** [review]

File: grunt/tasks/dist.js (hunk 3 above)
Comment (stephenwf) [reply grunt/tasks/dist.js:4]: The accessibility was testing many things, but not the HTML generated by the JavaScript. When running the "dist" task, I didn't want unnecessary warnings and errors that make it appear to have not worked.

The accessibility will still be run on `grunt test`

File: quilt/index.html/quilt.html (hunk 2 above)
Comment (stephenwf) [reply quilt/index.html/quilt.html:32]: Well spotted, how on earth did that happen, ill take another look at it

File: grunt/tasks/dist.js (hunk 3 above)
Comment (jamesgorrie) [reply grunt/tasks/dist.js:4]: Might as well remove it and leave a note if you think it's worth explaining.

**Pull request merged**